src/components/common/validation/time-format/ValidatedTimeEntry.jsx

```diff
@@ -8,7 +8,22 @@ const ValidatedTimeEntry = ({
   register,
   isRequired,
 }) => {
-  const errorMessage = `You must enter a ${timeType} in the HH:MM 24 hour clock format`;
+  const validateTime = (time) => {
+    if (time.length < 3 && time.length > 0) {
+      const timeRegEx = /^(\d|[01]\d|2[0-3])$/;
```

Comment:
Could these two regexes be given more descriptive names based on what they do/their purpose? 

Also, if they could be reused somewhere else in the application perhaps lifted out of here and moved into a shared location? 

If moved out it would be good to see some tests just for these regexes. It would help describe their purpose and what they do e.g. for someone fresh to the project/future maintainers of the project

Reply:
I've renamed the regexes, there isn't currently a need to use these elsewhere as this is the only place we need to validate times.